src/print.jl

```diff
@@ -103,7 +103,7 @@ math(s,mathmode) = mathmode ? s : "\$\$ $s \$\$"
 # helper to look up corresponding JuMPContainerData
 printdata(v::JuMPContainer) = _getmodel(v).varData[v]
 getname(x::JuMPContainer) = hasmeta(x, :model) ? printdata(x).name : "__anon__"
-function printdata(v::Array{Variable})
+function printdata(v::AbstractArray{Variable})
```

Comment:
Are you redefining printing for ``AbstractArray{Variable}``? If not, this doesn't need to be touched